libs/designsystem/src/lib/scss/base/_fonts.scss

```diff
@@ -0,0 +1,5 @@
+@import '~@fontsource/roboto/300.css';
+@import '~@fontsource/roboto/400.css';
+@import '~@fontsource/roboto/500.css';
+@import '~@fontsource/roboto/700.css';
+@import '~@fontsource/roboto/900.css';
```

Comment:
Should this just be included in `_typography.scss` instead? 🤔

Reply:
Can do - my tought was that definition of font-faces and typography are two separate things hence two separate files. But storing it in `_typography.scss` makes it easier to locate which font-faces are available when working in that file. And I guess that will be the file where you will most likely need that knowledge. So I will move it :D

Reply:
Resolved by 91b5b66